.github/workflows/build-and-test-multiplatform.yml

```
@@ -0,0 +1,794 @@
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  prepare-and-build-windows-x64:
+    name: Prepare and Build on Windows with MSVC (x64)
+    runs-on: windows-2022
+
+    steps:
+    # Step: Checkout the repository
+    - name: Checkout repository
```

yamllint

[warning] wrong indentation: expected 6 but found 4

(indentation)